src/TestTrait.php

```diff
@@ -36,25 +44,58 @@ public static function erisSetupBeforeClass()
      */
     public function erisSetup()
     {
-        $this->seedingRandomNumberGeneration();
-        $this->minimumEvaluationRatio(0.5);
+        $this->seed = intval(getenv('ERIS_SEED') ?: (microtime(true)*1000000));
+        if ($this->seed < 0) {
+            $this->seed *= -1;
+        }
+        $this->listeners = array_filter(
+            $this->listeners,
+            function ($listener) {
+                return !($listener instanceof MinimumEvaluations);
+            }
+        );
+        $tags = $this->getAnnotations();//from TestCase of PHPunit
+        $this->withRand($this->getAnnotationValue($tags, 'eris-method', 'rand', 'strval'));
+        $this->iterations = $this->getAnnotationValue($tags, 'eris-repeat', 100, 'intval');
+        $this->shrinkingTimeLimit = $this->getAnnotationValue($tags, 'eris-shrink', null, 'intval');
+        $this->listeners[] = MinimumEvaluations::ratio($this->getAnnotationValue($tags, 'eris-ratio', 50, 'floatval')/100);
+        $duration = $this->getAnnotationValue($tags, 'eris-duration', false, 'strval');
+        if ($duration) {
+            $this->limitTo(new DateInterval($duration));
+        }
```

Comment:
these are all defaults that can be extended or replaced in the test method by calling directly methods such as `withRand()`

Reply:
From my experience docs tend to lie because people forget to update them, so the priority should always be:
- code (i.e. method calls)
- method docs
- class docs

I hope that's ok?